src/components/modals/PlaceOrderModal.tsx

```diff
@@ -36,19 +36,18 @@ export const PlaceOrderModal: React.FC<PlaceOrderModalProps> = ({
           Order now
         </Button>
       </DialogTrigger>
-      <DialogContent className='md:w-9/10 md:max-h-dialog-content md:max-w-dialog-content h-full w-full max-w-full rounded-none sm:rounded-none md:flex md:gap-0 md:rounded-md md:border-0 md:p-0'>
-        <div className='md:w-image-container relative hidden md:block'>
+      <DialogContent className='md:w-9/10 h-full w-full max-w-full rounded-none sm:rounded-none md:flex md:max-h-dialog-content md:max-w-dialog-content md:gap-0 md:rounded-md md:border-0 md:p-0'>
+        <div className='relative hidden md:block md:w-image-container'>
```

Comment:
this is only code formatting (must have missed it previously)

Reply:
Got that . I think, that maybe the best way to  spot anything like this is to use a tailwind eslint plugin . Shall I add it? https://www.npmjs.com/package/eslint-plugin-tailwindcss?activeTab=readme

Reply:
that's up to you. I have a code format on save, so either I didn't do ctrl+S and Prettier didn't run previous time I was working on this file, or Prettier decided to rearrange classes this time. I thought I'd comment on it to save you some time, trying to figure out why was this change done ;)